### PR TITLE
task 5 (окремо 5-та домашка)

### PR DESCRIPTION
**Pull request by demydenko**

### CODE REVIEW

File: students/demydenko/05-bag-of-words/baseline.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+vectorizer = DictVectorizer()
+
+
+# vec_train_tokens = vectorizer.fit(x_train)
+vec_train_tokens_transformed = vectorizer.fit_transform(x_train)
+# print("\nTotal number of features: ", len(vec_train_tokens.get_feature_names()))
+
+vec_test_tokens = vectorizer.transform(x_test)
+
+clf = LogisticRegression()
```
Comment (mariana-scorp) [inline students/demydenko/05-bag-of-words/baseline.py]: Тут теж напишу: задавай, будь ласка, `random_state`. Тобі ж простіше буде порівнювати результати. Інакше accuracy може в межах 1-2 відсотків варіюватись.

File: students/demydenko/05-bag-of-words/data_loader.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+
+# завантаження даних, попередня обробка і розбивка та train test
+def load_and_cook_data(load_from_file=False):
+    reviews_uk = load_data(load_from_file=load_from_file)
+    df_ = pd.DataFrame(reviews_uk)
+    df_ = df_.drop_duplicates()
+
+    #  питання ігноруємо!
+    df_without_q = df_[df_['text'].str.find("?") == -1]
```
Comment (mariana-scorp) [inline students/demydenko/05-bag-of-words/data_loader.py]: 👍

File: students/demydenko/05-bag-of-words/data_loader.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        df_['fn_' + f] = df_['text'].str.contains(f, flags=re.IGNORECASE, regex=True)
+    df_['fn_minus_len'] = len(df_['text_minus'])
+    df_["rating"] = pd.to_numeric(df_["rating"])
+    df_["fp_plus_len"] = pd.to_numeric(df_["fp_plus_len"])
+    df_["fn_minus_len"] = pd.to_numeric(df_["fn_minus_len"])
+
+    df_y = df_['rating']
+    df_y = pd.DataFrame(df_y)
+
+    # спрощуємо систему оцінок
```
Comment (mariana-scorp) [inline students/demydenko/05-bag-of-words/data_loader.py]: Добре, що розбив на два класи 👍
